app/services/RegistrationServiceImpl.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import models.requests.RegistrationRequest
+import models.{InsertResult, Registration}
+import repositories.RegistrationRepository
+import uk.gov.hmrc.domain.Vrn
+
+import java.time.Clock
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.Future
+
+@Singleton
+class RegistrationServiceImpl @Inject()(
```

Comment:
Maybe rename to RegistrationServiceRepositoryImpl or RegistrationServiceMongoImpl to match the format of the format of the other, and make clear what each is doing?